share/completions/kitty.fish

```diff
@@ -1,6 +1,7 @@
-function __kitty_completions
-    # Send all words up to the one before the cursor
-    commandline -cop | kitty +complete fish
+function __ksi_completions
```

Comment:
Can we keep the "kitty" name?

I quite dislike the "ksi" abbreviation here and it has confused users in the past.

Reply:
Oh, wait, this is a vendored thing.

Ideally packages would simply install their completions instead of us requiring to ship guff like this, but in this case it's probably okay.